config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users
+  resources :posts
+  root 'posts#index'
```

Comment:
Please limit the posts routes with `resources :posts, only: [:new, :create, :index]`